### PR TITLE
Fixing broken links to ScyllaDB University lessons, Scylla University…

### PR DESCRIPTION
**Pull request by guy9**
… -> ScyllaDB University

### CODE REVIEW

**Comment by guy9** [comment]
Fixes https://github.com/scylladb/scylladb/issues/14350#issuecomment-1602145493
@annastuchlik please review

File: docs/operating-scylla/procedures/backup-restore/index.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ Procedures to backup and restore your Scylla data safely
 
 * :doc:`Delete your Backup </operating-scylla/procedures/backup-restore/delete-snapshot/>`
 
-* Also covered in the `Backup and Restore lesson <https://university.scylladb.com/courses/the-mutant-monitoring-system-training-course/lessons/backup-and-restore/>`_ on Scylla University.
+* Also covered in the `Cluster Backup Using ScyllaDB Manager lesson <hhttps://university.scylladb.com/courses/scylla-operations/lessons/scylla-manager-repair-and-tombstones/topic/cluster-backup-using-scylla-manager/>`_ on ScyllaDB University.
```
Comment (annastuchlik) [inline docs/operating-scylla/procedures/backup-restore/index.rst:29]:
```suggestion
* Also covered in the `Cluster Backup Using ScyllaDB Manager lesson <https://university.scylladb.com/courses/scylla-operations/lessons/scylla-manager-repair-and-tombstones/topic/cluster-backup-using-scylla-manager/>`_ on ScyllaDB University.
```

**Review by annastuchlik — CHANGES_REQUESTED** [review]
A minor change is required to make one of the links work.

**Comment by guy9** [comment]
Thanks @annastuchlik , fixed.

**Review by annastuchlik — APPROVED** [review]
LGTM

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please merge.

**Pull request merged**